pkg/karmadactl/cmdinit/utils/examples.go

```diff
@@ -206,11 +206,15 @@ Step 2: Show members of karmada
 
 Register cluster with 'Pull' mode
 
-Step 1: Use "%[2]s register" command to register the cluster to Karmada control plane. "--cluster-name" is set to cluster of current-context by default.
-(In member cluster)~# %[2]s%[3]s
+Step 1: Create bootstrap tokens and get the full '%[2]s register' flag needed to register the member cluster using the token.
+(In karmada)~# %[2]s token create --print-register-command --kubeconfig %[1]s/karmada-apiserver.config
+%[2]s register [karmada-apiserver-endpoint] --token [token] --discovery-token-ca-cert-hash [ca-cert-hash]
 
-Step 2: Show members of karmada
-(In karmada)~# kubectl --kubeconfig %[1]s/karmada-apiserver.config get clusters
+Step 2: Use the output result from step 1 to register the cluster to Karmada control plane. "--cluster-name" is set to cluster of current-context by default.
+(In member cluster)~# %[2]s register [karmada-apiserver-endpoint] --token [token] --discovery-token-ca-cert-hash [ca-cert-hash]
+
+Step 3: Show members of karmada
+(In karmada)~# karmadactl --kubeconfig %[1]s/karmada-apiserver.config get clusters
```

Comment:
Something like:
```
Step 1: Create bootstrap token and generate the '%[2]s register' command which will be used later.
~# %[2]s token create --print-register-command --kubeconfig=<path-to-karmada-config>
This command will generate a registration command similar to:

%[2]s register 172.18.0.5:5443 --token t8xfio.640u9gp9obc72v5d --discovery-token-ca-cert-hash sha256:9cfa542ff48f43793d1816b1dd0a78ad574e349d8f6e005e6e32e8ab528e4244

Step 2: Use the output from Step 1 to register the cluster to the Karmada control plane. 
You need to specify the target member cluster by flag '--kubeconfig'
~# %[2]s register 172.18.0.5:5443 --token t8xfio.640u9gp9obc72v5d --discovery-token-ca-cert-hash sha256:9cfa542ff48f43793d1816b1dd0a78ad574e349d8f6e005e6e32e8ab528e4244 --kubeconfig=<path-to-member-cluster-kubeconfig>

Step 3: Show members of Karmada
~# %[2]s --kubeconfig=<path-to-karmada-config> get clusters

The %[2]s register command has several optional parameters for setting the properties of the member cluster. For more details, run:

~# %[2]s register --help
```

Reply:
Since this is an example of `karmadactl init`, `path-to-karmada-config` is deterministic and can be referred to directly by `%[1]s/karmada-apiserver.config`